subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -435,6 +434,9 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id)
 	cig->c_latency = c_max_latency;
 	cig->p_latency = p_max_latency;
 
+	/* Indicates number of allocated CISes throughout CIG lifetime */
+	cig->lll.num_cis = cis_count;
+
```

Comment:
Have you tested multiple CIS create? I see assertion here:

https://github.com/zephyrproject-rtos/zephyr/pull/55845/files#diff-89b902b4402e615de7b88e8bf0c3216c8eeff7c495ff7208a9fed9c8271994acR644

You cannot reuse existing code in `ull_conn_iso.c` to use `cig->lll.num_cis` if it is always total cis_count for Central and active CISes for Peripheral. Upstream code represents active CIS count in `cig->lll.num_cis` for both Central (`cig->cis_count` as total) and Peripheral.

Reply:
Yes I have tested with 2x CISes. Create and connect two CISes, disconnect first, reconnect, disconnect both, delete CIG. The reason the existing code can be used is by checking the `active` flag.
I don't see any asserts, so can it possibly be due to other changes?

Reply:
We are 3 weeks behind compared to upstream Zephyr, but I don't remember seeing any breaking changes lately.

Reply:
Also, the only way I can see that assert could happen, is if the `cig->lll.num_cis` was incorrectly incremented, and with my code it would always be the same value for central.

Reply:
This is the line that would set `cig->lll.num_cis` to > 1 i.e. `cig->lll.num_cis = cis_count;` while CISes are created sequentially on each ACL or same ACL. Each CIS Create Control Procedure would be few connection events apart, it should be hitting the assert in `ull_conn_iso_ticker_cb()`, not sure why you do not see it.

My work rebased on this PR is here: https://github.com/zephyrproject-rtos/zephyr/pull/55924

You can reproduce the assertion by running  `tests/bluetooth/bsim/ll/cis/tests_scripts/connected_iso.sh`

![image](https://user-images.githubusercontent.com/6350656/225904144-4b063765-8953-46ee-a4fd-94142ca6f6e0.png)

Reply:
Ok, ignore it. It is a change in my PR that causes the assert.

Reply:
I have updated https://github.com/zephyrproject-rtos/zephyr/pull/55924. This is passing now.